libs/@local/hash-isomorphic-utils/src/save.ts

```diff
@@ -505,6 +508,17 @@ export const save = async (
     >({
       variables: { entityId: blockCollectionEntityId, actions },
       mutation: updateBlockCollectionContents,
+      refetchQueries: [
+        {
+          query: getEntityQuery,
+          variables: {
+            entityId: blockCollectionEntityId,
+            ...zeroedGraphResolveDepths,
+            hasLeftEntity: { incoming: 2, outgoing: 2 },
+            hasRightEntity: { incoming: 2, outgoing: 2 },
```

Comment:
Comment why these depths please... if it's important that these variables are the same as in the other place we have them, maybe better to share them rather than relying on keeping them in sync

Reply:
The comment indicates that the required depths are:
```
            hasLeftEntity: { incoming: 2, outgoing: 0 },
            hasRightEntity: { incoming: 0, outgoing: 2 },
```
so at the moment we're overfetching data (things that link _to_ the page rather than being linked from it), unless there's a reason we need such links